test/browser_tests/cucumber-new/features/suites/apps/ask_cfpb.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Test the Ask CFPB page in English and Spanish
+  As a public visitor to the consumerfinance.gov site
+  I want to search using the Ask CFPB page in English and Spanish
+  So that I can verify the correct results are returned
+
+@smoke_testing @ask_cfpb 
+Scenario Outline: Make sure incorrect slug in URL is corrected
+  Given I visit the incorrect "www.consumerfinance.gov/<bad_url>" URL
+  Then I should be directed to "www.consumerfinance.gov/<correct_url>" URL
+  And I should see the page title contains "<title>"
+
+Examples:
+| bad_url                                           | correct_url                           | title                    |
+| ask-cfpb/1163/what-wire-transfer_THIS_IS_Bad.html | ask-cfpb/1163/what-wire-transfer.html | What is a wire transfer? |
+| ask-cfpb/1065/what-is-an-ach_THIS_IS_BAD.html     | ask-cfpb/1065/what-is-an-ach.html     | What is an ACH?          |
+| es/obtener-respuestas/c/manejar-una-cuenta-bancaria/983/como-suspendo-el-pago-de-un-cheque_BAD.html | es/obtener-respuestas/c/manejar-una-cuenta-bancaria/983/como-suspendo-el-pago-de-un-cheque.html | ¿Cómo suspendo el pago de un cheque? |
+| es/obtener-respuestas/c/obtener-una-tarjeta-de-credito/47/que-es-un-periodo_BAD.html                | es/obtener-respuestas/c/obtener-una-tarjeta-de-credito/47/que-es-un-periodo-de-gracia-como-funciona.html | Qué es un período de gracia |
+
+@smoke_testing @ask_cfpb
+Scenario Outline: In Ask CFPB, verify that the search box contains a sample question and search button is disabled by default
```

Comment:
The search boxes no longer have sample questions. Remove this scenario.

Reply:
Remove this scenario.